rails_app/spec/adapters/graphite_adapter_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe GraphiteAdapter do
+  before(:each) do
+    allow(api_mock).to receive(:metrics)
+    expect(GraphiteAPI).to receive(:new)
+      .with(graphite: 'localhost:2003', prefix: 'price_monitor_internal')
+      .and_return(api_mock)
+    @graphite = GraphiteAdapter.new
+  end
+
+  let(:api_mock) { double :api_mock }
+
+  describe '.initialize' do
+    it 'instantiate GraphiteAdapter object and check if default values were loaded' do
```

Comment:
'instantiate GraphiteAdapter object' i think its enought

Reply:
Explicit is better than implicit.